components/camel-dhis2/camel-dhis2-component/src/main/java/org/apache/camel/component/dhis2/Dhis2Configuration.java

```diff
@@ -30,12 +30,16 @@ public class Dhis2Configuration {
     @UriParam(description = "DHIS2 server base API URL (e.g., https://play.dhis2.org/2.39.1.1/api)")
     private String baseApiUrl;
 
-    @UriParam(description = "DHIS2 account username for accessing the DHIS2 API", secret = true, label = "security")
+    @UriParam(description = "Username of the DHIS2 user to operate as", secret = true, label = "security")
     private String username;
 
-    @UriParam(description = "DHIS2 account password for accessing the DHIS2 API", secret = true, label = "security")
+    @UriParam(description = "Password of the DHIS2 username", secret = true, label = "security")
     private String password;
 
+    @UriParam(description = "Personal access token to authenticate with DHIS2. This option is mutually exclusive to username and password",
+              secret = true, label = "security")
+    private String pat;
```

Comment:
Can you rename this to `personalAccessToken` as pat is too short and confusing what it means